transmart-core-db/src/main/groovy/org/transmartproject/db/multidimquery/DimensionImpl.groovy

```diff
@@ -420,8 +420,10 @@ class PatientDimension extends I2b2Dimension<Patient, Long> implements Composite
     List elemFields = ["id", "trial", "inTrialId", "birthDate", "deathDate",
                       "age", "race", "maritalStatus", "religion",
                       new PropertyImpl('sex', 'sex', String) {
+                          @Override def get(element) { super.get(element).toString() }},
+                      new PropertyImpl('sexCd', 'sexCd', String) {
                           @Override def get(element) { super.get(element).toString() }
```

Comment:
@ewelinagr  You don't need a new PropertyImpl for this. If you just specify the property as string, the serialization will take that property as-is. You only need to create a custom PropertyImpl if it is not a plain property, e.g. the 'sex' is an enum that needs to be converted to string.